### DISCUSSION TITLE
EmuDeck

### DISCUSSION THREAD

**Discussion by abueide** [Jovian-Experiments/Jovian-NixOS#200] [Q&A] [answered]
I'm really interested in trying out nixos on the deck my only reservation is whether emudeck would work well on it? Has anyone tried it out that can offer some insight?

**Comment by K900** [comment, marked as answer]
No, it won't. It expects a conventional distribution, with all that entails. You can, however, manually install the emulators you want from nixpkgs.

**Reply by K900** [reply]
Also, please use discussions for support questions.
